### PR TITLE
feat(cyod): iOS signing certificates for org and project

### PR DESCRIPTION
**Pull request by ssingh55**
4/7 in the CYOD stack. Bases on #1714 (cyod-03-org-registration-toggle). Merge in order 01→07.

Org- and project-level iOS signing certificates: the add-certificate drawer, the existing-certificates tab, and delete confirmation. Also wires both this and 03's toggle into the organization settings page.

Design review applied: `py-2 px-4` drawer body, dark dividers, tip callout at 4px radius on a 4px accent bar, medium input labels, italic "No file chosen", selected files as removable chips, and the certificate card ramp from the Figma frames (name 700, meta labels 12px/500, values 12px/600).

cc @Yibaebi — the delete dialog and the card ramp are the two worth a close look.

### CODE REVIEW

File: app/components/organization/settings/index.hbs [hunk 1]
```diff
@@ -22,6 +22,20 @@
 
   <AkDivider class='my-3' @color='dark' />
 
+  {{#if (and this.me.org.is_owner this.organization.isCyodEnabled)}}
```
Comment (SmitGala) [inline app/components/organization/settings/index.hbs]: can we make getter ?

File: app/components/organization/settings/index.hbs [hunk 2]
```diff
@@ -22,6 +22,20 @@
...
+    <Organization::DeviceRegistration />
+
+    <AkDivider class='my-3' @color='dark' />
+
+    {{! Certificates are part of the CYOD setup, so they collapse with it.
+    Guarded here as well as in the component so the divider goes too. }}
+    {{#if this.organization.isCyodRegistrationEnabled}}
+      <Organization::SigningCertificate />
+
+      <AkDivider class='my-3' @color='dark' />
```
Comment (SmitGala) [inline app/components/organization/settings/index.hbs]: if condition only for component not for Divider

File: app/components/organization/signing-certificate/index.hbs [hunk 3]
```diff
@@ -0,0 +1,623 @@
+{{#if this.visible}}
+  <div
+    local-class='signing-cert-container'
```
Comment (SmitGala) [inline app/components/organization/signing-certificate/index.hbs]: Akstack 
width=full

or class= 'w-full'

File: app/components/organization/signing-certificate/index.hbs [hunk 4]
```diff
@@ -0,0 +1,623 @@
...
+  <AkDrawer
+    @open={{this.showDrawer}}
+    @onClose={{this.handleClose}}
+    @anchor='right'
+    as |dr|
+  >
+    <div
+      data-test-orgSigningCert-drawer
+      local-class='cert-drawer'
+      {{style height='100%'}}
```
Comment (SmitGala) [inline app/components/organization/signing-certificate/index.hbs]: can we add this style in cert-drawer class?

File: app/components/organization/signing-certificate/index.hbs [hunk 5]
```diff
@@ -0,0 +1,623 @@
...
+          >
+            <AkStack @direction='column' @spacing='0.5'>
+              <AkTypography data-test-orgSigningCert-deleteConfirmTitle>
+                {{t 'orgSigningCertDeleteQuestion'}}
+                <strong>{{if
+                    this.pendingDelete.name
+                    this.pendingDelete.name
+                    (t 'orgSigningCertUnnamed')
+                  }}?</strong>
+              </AkTypography>
```
Comment (SmitGala) [inline app/components/organization/signing-certificate/index.hbs:105]: Format properly

File: app/components/organization/signing-certificate/index.hbs [hunk 6]
```diff
@@ -0,0 +1,623 @@
...
+              @buttonVariant={{true}}
+              @isActive={{eq this.activeTab 'existing'}}
+              @onTabClick={{fn this.handleTabChange 'existing'}}
+            >
+              {{t 'orgSigningCertExistingTab'}}
+            </Akt.tabItem>
+          </AkTabs>
+        </div>
+
+        {{#if (eq this.activeTab 'add')}}
```
Comment (SmitGala) [inline app/components/organization/signing-certificate/index.hbs]: please use getter for 

and , eq, etc

File: app/components/organization/signing-certificate/index.scss [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+
+.cert-meta {
+  display: grid;
+  grid-template-columns: auto 1fr;
+  column-gap: 12px;
+  row-gap: 6px;
+  align-items: baseline;
+}
+
+.drawer-body > * {
```
Comment (SmitGala) [inline app/components/organization/signing-certificate/index.scss]: Why > * was done?

File: app/components/organization/signing-certificate/index.scss [hunk 8]
```diff
@@ -0,0 +1,129 @@
...
+  padding: 1.14286em;
+}
+
+.file-name {
+  word-break: break-all;
+}
+
+.no-file-chosen {
+  font-style: italic;
+}
```
Comment (SmitGala) [inline app/components/organization/signing-certificate/index.scss]: For single style ...use inline css

File: app/components/organization/signing-certificate/index.scss [hunk 9]
```diff
@@ -0,0 +1,129 @@
...
+    var(--organization-signing-certificate-file-chip-border-color);
+  border-radius: var(
+    --organization-signing-certificate-file-chip-border-radius
+  );
+  background: var(--organization-signing-certificate-file-chip-background);
+}
+
+.file-chip .file-name {
+  font-size: 1rem;
+}
```
Comment (SmitGala) [inline app/components/organization/signing-certificate/index.scss]: is this really required?

File: app/components/organization/signing-certificate/index.ts [hunk 10]
```diff
@@ -0,0 +1,349 @@
...
+    return this.certs.length > 0;
+  }
+
+  certStatusColor = (cert: CertInfo) => (cert.is_expired ? 'error' : 'success');
+
+  // The backend refuses to delete the active cert only while siblings exist —
+  // there would be no fallback to promote. A lone cert is deletable even though
+  // it is always the active one, so the org can clear its setup entirely.
+  isDeleteDisabled = (cert: CertInfo) =>
+    !!cert.is_active && this.certs.length > 1;
```
Comment (SmitGala) [inline app/components/organization/signing-certificate/index.ts]: What is this?

use getter instead

File: app/components/organization/signing-certificate/index.ts [hunk 11]
```diff
@@ -0,0 +1,349 @@
...
+
+  get hasCert() {
+    return !!(this.cert && this.cert.team_id);
+  }
+
+  get hasCerts() {
+    return this.certs.length > 0;
+  }
+
+  certStatusColor = (cert: CertInfo) => (cert.is_expired ? 'error' : 'success');
```
Comment (SmitGala) [inline app/components/organization/signing-certificate/index.ts]: What is this?

use getter instead

File: app/components/organization/signing-certificate/index.ts [hunk 12]
```diff
@@ -0,0 +1,349 @@
...
+  clearP12() {
+    this.p12File = null;
+  }
+
+  @action
+  clearProfile() {
+    this.profileFile = null;
+  }
+
+  resetForm() {
```
Comment (SmitGala) [inline app/components/organization/signing-certificate/index.ts:243]: add @ action

File: app/components/organization/signing-certificate/index.ts [hunk 13]
```diff
@@ -0,0 +1,349 @@
...
+
+  @action
+  requestDelete(cert: CertInfo) {
+    this.pendingDelete = cert;
+  }
+
+  @action
+  cancelDelete() {
+    this.pendingDelete = null;
+  }
```
Comment (SmitGala) [inline app/components/organization/signing-certificate/index.ts]: Action should be moved before task

File: app/styles/_component-variables.scss [hunk 14]
```diff
@@ -3002,6 +3002,43 @@ body {
...
+  --organization-signing-certificate-empty-state-block-border-radius: var(
+    --border-radius
+  );
+  --organization-signing-certificate-file-chip-background: var(--common-white);
+  --organization-signing-certificate-file-chip-border-color: var(
+    --divider-dark
+  );
+  --organization-signing-certificate-file-chip-border-radius: var(
+    --border-radius-8
+  );
```
Comment (SmitGala) [inline app/styles/_component-variables.scss]: divider-dark has been defined 3-4 times
check for another var as well

make sure component dont have multiple same variable

File: translations/ja.json [hunk 15]
```diff
@@ -1930,6 +1930,7 @@
   "orgSigningCertDesc": "iOS署名証明書（.p12 + プロビジョニングプロファイル）をアップロードして、自社デバイスでの動的スキャン用にアプリを再署名します。設定すると、組み込みのワイルドカードプロファイルの代わりに使用されます。",
   "orgSigningCertDescProject": "このプロジェクト専用のiOS署名証明書をアップロードします。このプロジェクトの動的スキャンでは、組織の証明書よりも優先されます。",
   "orgSigningCertNone": "証明書を追加すると、ここに詳細が表示されます。",
+  "orgSigningCertNoneTitle": "既存の証明書がありません",
```
Comment (SmitGala) [inline translations/ja.json]: Verify translation

File: app/components/organization/signing-certificate/index.ts [hunk 16]
```diff
@@ -0,0 +1,349 @@
...
+    }
+  });
+
+  // Org scope: mark one cert active (the backend clears the previous active).
+  activateCert = task({ restartable: true }, async (cert: CertInfo) => {
+    this.busyCertId = cert.id ?? null;
+    try {
+      await this.ajax.post(`${this.baseUrl}${cert.id}/activate/`, {});
+      this.notify.success(this.intl.t('orgSigningCertActivated'));
+      await this.load.perform();
```
Comment (SmitGala) [inline app/components/organization/signing-certificate/index.ts]: give proper line spacing

File: app/components/organization/signing-certificate/index.ts [hunk 17]
```diff
@@ -0,0 +1,349 @@
...
+  });
+
+  // Org scope: delete one cert by id. The backend returns 409 for the active
+  // cert while siblings exist — surfaced as an error toast.
+  deleteOrgCert = task({ restartable: true }, async (cert: CertInfo) => {
+    this.busyCertId = cert.id ?? null;
+    try {
+      await this.ajax.delete(`${this.baseUrl}${cert.id}/`);
+      this.notify.success(this.intl.t('orgSigningCertDeleted'));
+      await this.load.perform();
```
Comment (SmitGala) [inline app/components/organization/signing-certificate/index.ts:352]: give proper line spacing

File: app/components/organization/signing-certificate/index.ts [hunk 18]
```diff
@@ -0,0 +1,349 @@
...
+    } else {
+      await this.deleteOrgCert.perform(cert);
+    }
+
+    this.pendingDelete = null;
+  });
+
+  deleteCert = task({ restartable: true }, async () => {
+    try {
+      await this.ajax.delete(this.baseUrl);
```
Comment (SmitGala) [inline app/components/organization/signing-certificate/index.ts:332]: add line below

File: app/components/organization/signing-certificate/index.hbs [hunk 19]
```diff
@@ -0,0 +1,623 @@
...
+                    </AkStack>
+
+                    <AkStack @spacing='1' @alignItems='center'>
+                      {{#if
+                        (and
+                          this.canActivate
+                          (not cert.is_active)
+                          (not cert.is_expired)
+                        )
+                      }}
```
Comment (SmitGala) [inline app/components/organization/signing-certificate/index.hbs]: add getter for this

**Review by SmitGala — CHANGES_REQUESTED** [review]
